src/dmd/globals.d

```diff
@@ -118,6 +118,7 @@ extern (C++) struct Param
     bool vgc;               // identify gc usage
     bool vfield;            // identify non-mutable field variables
     bool vcomplex = true;   // identify complex/imaginary type usage
+    bool vin;               // identify 'in' parameters
```

Comment:
(frontend) headers

Reply:
Should be done. For reference, it's still a pain to run the command on OSX because there are a bunch of unrelated changes.